src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -98,8 +95,7 @@ services:
 
     packagist.menu.user:
         class: Knp\Menu\MenuItem
-        factory_service: packagist.menu_builder
-        factory_method: createUserMenu
+        factory: [@packagist.menu_builder, createUserMenu]
```

Comment:
Please quotes the string starting with ``@`` to be valid Yaml (see the 2.8 deprecations)